### PR TITLE
 docs(readme): Updated README with ergonomic changes

### PR DESCRIPTION
**Pull request by EricBrianAnil**
## Description

Added helpful backlinks to 
1. BPMN 2.0
2. Docker
3. Kubernetes
4. Kafka Connector of zeebe

## Related issues

closes #

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ x] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3ba277cdc6d0862173cfd3fa07274add0645dad4
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Review by jwulf — CHANGES_REQUESTED** [review]
Thanks for the PR @EricBrianAnil - this is a good ergonomic addition to the docs. 

One thing about the commit, you'll need to amend the commit message, following these guidelines: https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3ba277cdc6d0862173cfd3fa07274add0645dad4
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3ba277cdc6d0862173cfd3fa07274add0645dad4
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by EricBrianAnil** [comment]
> Thanks for the PR @EricBrianAnil - this is a good ergonomic addition to the docs.
> 
> One thing about the commit, you'll need to amend the commit message, following these guidelines: https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines

Amended the commit message following the guidelines. Also added the Apache license badge with commit 2157fda3a7d69526da8226b9c91fad77709df26a

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3ba277cdc6d0862173cfd3fa07274add0645dad4
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by jwulf** [comment]
I think the description in the commit message needs to be all lower-case. Gitcop will tell you if it passes. If doing it all lower-case doesn't work, then I'd go with: `chore(broker): update README with links`.

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ By participating, you are expected to uphold this code. Please [report](https://
 unacceptable behavior as soon as possible.
 
 ## License
-
+ [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```
Comment (korthout) [inline README.md:58]: Hi @EricBrianAnil, thanks for this PR. Could you please explain why you've added this line?

**Comment by korthout** [comment]
> I think the description in the commit message needs to be all lower-case. Gitcop will tell you if it passes. If doing it all lower-case doesn't work, then I'd go with: `chore(broker): update README with links`.

@jwulf The problem here is that the PR still contains all previously added commits as well. 

@EricBrianAnil Please rebase (squash) your commits to 1 commit that matches the format requested by GitCop and just force push that commit.

**Comment by jwulf** [comment]
Hi @EricBrianAnil, did you see the comments from @korthout? Do you need any help with this?

**Comment by EricBrianAnil** [comment]
> Hi @EricBrianAnil, did you see the comments from @korthout? Do you need any help with this?

Heyy, yes. Sorry for the late reply, my laptop is undergoing some repair. I will get to it as soon as poss.

File: README.md (hunk 1 above)
Comment (EricBrianAnil) [reply README.md:58]: Yess sure, It's the License badge. I've noticed that most official repos have the badge that redirects it to the original Apache License file. so yea :p

**Comment by korthout** [comment]
Hi @EricBrianAnil, please let me know if you need any help with this.

File: README.md (hunk 1 above)
Comment (korthout) [reply README.md:58]: Please remove this line, our license section explains that we use different licenses for different parts of the code, so not just the apache license.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fe8969a169e5336d9dd157fe8a6076cd9c124a70
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Review by korthout — APPROVED** [review]
@EricBrianAnil I've rebased your commits into 1, updated the commit message, removed the license badge and altered the link for kafka-connect-zeebe to kafka.apache.org (because kafka-connect-zeebe is a community project).

**Comment by korthout** [comment]
ping @jwulf

**Review by jwulf — APPROVED** [review]
LGTM

**Pull request merged**